go.mod

```diff
@@ -8,11 +8,10 @@ require (
 	github.com/minio/sha256-simd v1.0.0 // indirect
 	github.com/philhofer/fwd v1.1.1 // indirect
 	github.com/prometheus/procfs v0.7.3
+	github.com/minio/pkg v1.0.11
```

Comment:
@donatello you cannot import `minio/pkg` `minio/pkg` is AGPLv3 and `minio/madmin-go` is Apache they are incompatible for linkage i.e basically madmin-go would AGPLv3 which will lead to consumers of this SDK to be bounded by AGPLv3.

My suggestion is to avoid the usage and find replacements locally - madmin-go generally should be self-contained.

Reply:
Ah ok, let me fix this.

Reply:
Done!